importDBdmpFile/1_runDbUpdate.sh

```diff
@@ -1,30 +1,30 @@
 #!/bin/bash
 
-# Script for downloading the gocdb5 db dmp file and importing into the 
-# local oracle xe instance using sqlplus. 
-# Notes: 
-# - The updatelog.txt file is appended each time with log messages whenever the script is ran. 
+# Script for downloading the gocdb5 db dmp file and importing into the
```

Comment:
I see that there is a PR #23  addressing whitespaces. Is it being repeated here ?